Source/public/views/top-navigation.html

```diff
@@ -1,11 +1,15 @@
 <header>
-    <nav class="navbar navbar-expand-md navbar-dark fixed-top" style="background-color: #121212;  border-bottom: 2px solid #DC4405;">
-        <a class="navbar-brand" href="#" style="width: 90px; font-size: 80%; font-weight: bold;">OSU Energy Dashboard</a>
+    <nav class="navbar navbar-expand-md navbar-dark fixed-top border-bottom-orange" style="background-color: #121212;">
+		<div style="width: 150px; font-family: 'Stratum2-Bold';">
+			<a class="navbar-brand" href="#" style="font-size: 80%; font-weight: bold; position: absolute; top: 0%; left: 10px;">OSU</a>
```

Comment:
Why are these a tags and not <span>s?

Reply:
Just because thats what it was previously. I see a tags generally used for links and span seems to be for changing text color? Just what came up from google. And they are supposed to link to home page.

Reply:
Ohhhh, I didnt see they were linking to the home page... right right!